guides/common/modules/proc_installing-capsule-server-packages.adoc

```diff
@@ -4,7 +4,6 @@
 Before installing {SmartProxyServer} packages, you must update all packages that are installed on the base operating system.
 
 ifdef::foreman-el,katello,satellite[]
-* xref:el-7-package[{EL} 7]
 * xref:el-8-package[{EL} 8]
```

Comment:
If there's just a single item in the list, does it make sense to keep it in? I'd drop it.

Reply:
Addressed :+1:

Reply:
Perhaps it was lost, because it's still a listing.

Reply:
Addressed :+1: